### ISSUE TITLE
Changes in the repository structure and other modifications

### ISSUE THREAD

**Issue by rvicedomini** [CamilaDuitama/muset#9]
Hi @CamilaDuitama & @frankandreace ,

I want to mention that I did some important changes on the repository.
- kmat_tools executable does not change name, only the main pipeline script (I discussed about it with Rayan last Friday)
- muset already is in the bin folder, kmat_tools will be placed there automatically after building
- I made muset help message style consistent with kmat_tools
- muset executable should be run as it is, without the "pipeline" command.
- I created a simpler conda recipe in `conda/muset` (it is not needed to create a full copy of the sources to build a conda package!). To build the package run `conda-build conda/muset` from the root of the project. @CamilaDuitama you should probably delete the "muset" conda packages you built previously and rebuild one from scratch.
- I simplified the `environment.yaml`
- I modified a bit the singularity recipe, removing unnecessary lines or generated files. Moreover I fixed an issue I had with running ggcat from the container.
- I removed the `test/` path prefix from the `fof.txt` file. I think muset should be run from the base `test` directory to avoid "polluting" the main repository directory. I don't know... we might even consider to rename it `example`... not a pressing matter, though.

I don't know if there is more to add... if there is, I'll add it later...

**Comment by rvicedomini** [comment]
I've did some modifications related to the `-s` option. More precisely, I replaced the `-s` option with the `-i` option which accepts a file path to an existing matrix (before, `-s` assumed the matrix had a fixed name within the output folder).
Now you can use a previously computed input matrix and make the pipeline work in a different output directory (if desired).

One thing I'd like to do is to avoid requiring the positional argument (fof file) when the input matrix is provided through `-i`, as it is only required by kmtricks (which is skipped).

**Comment by rvicedomini** [comment]
I just committed an update that ignores the positional argument if present when `-i` option is provided.

**Comment by rvicedomini** [comment]
@CamilaDuitama & @frankandreace, I was looking at the code and noticed that the last part of the pipeline is quite inefficient for two main reasons. First, sorting does not seem to be needed (also considering that ggcat output is not deterministic). Second, for each unitig ID, a linear scan of the unitig FASTA is done to find its sequence (which makes this step quadratic in the number of unitigs).

If all we want is to have, as final output, a unitig matrix containing the actual sequence in the first column (instead of the ID), I only have to change one line of code for the command `kmat_tools unitig` (thus skipping the aforementioned part). I could also add a flag if you want the user to decide whether to have the ID or the sequence in the first column.

**Comment by CamilaDuitama** [comment]
Yes please, if it's more easily done from the kmat_tools unitig code then it's much better. Thanks @rvicedomini

**Comment by rvicedomini** [comment]
Flag `-s` has been added to both `muset` and `kmat_tools unitig` commands in order to write unitig sequence in the output matrix instead of the ID (default behavior when the flag is not used).

I also made a small change to the main script. More precisely, I now update the PATH environment variable at the beginning using the directory of the script. In this way, the `kmat_tools` executable which is called should be the one within the same folder of `muset`.
